wiki/Submission/id.md

```diff
@@ -0,0 +1,27 @@
+# Submission
```

Comment:
would be better if the article's title could be translated as well i guess into ``Submisi`` or ``Submisi Beatmap`` (as it has been the case with many other translated articles out there)